cylc/uiserver/tests/conftest.py

```diff
@@ -360,3 +363,51 @@ def workflow_run_dir(request):
     yield flow_name, log_dir
     if not request.session.testsfailed:
         rmtree(run_dir)
+
+@pytest.fixture
+def mock_glbl_cfg(tmp_path: Path, monkeypatch: pytest.MonkeyPatch):
```

Comment:
Can you not import this fixture from Cylc? Redefining it seems like duplication.

Reply:
tldr; no, not easily.